node/service/src/lib.rs

```diff
@@ -1235,11 +1288,16 @@ pub fn new_chain_ops(
 > {
 	config.keystore = service::config::KeystoreConfig::InMemory;
 
+	// FIXME no overseer is ever constructed here
```

Comment:
unresolved

Reply:
It's actually necessary for the `RelayChainSelection` construction.